brage-import/src/main/java/no/sikt/nva/brage/migration/lambda/BrageEntryEventConsumer.java

```diff
@@ -71,22 +64,11 @@ public class BrageEntryEventConsumer implements RequestHandler<S3Event, Publicat
     private static final String S3_URI_TEMPLATE = "s3://%s/%s";
     private static final String ERROR_SAVING_BRAGE_IMPORT = "Error saving brage import for record with object key: ";
     private static final Logger logger = LoggerFactory.getLogger(BrageEntryEventConsumer.class);
-    private static final String RESOURCES = "resources";
-    private static final String SEARCH = "search";
-    private static final String DOI = "doi";
-    private static final String APPLICATION_JSON = "application/json";
-    public static final String TITLE = "title";
-    public static final String CONTEXT_TYPE = "contextType";
-    public static final String AGGREGATION = "aggregation";
-    public static final String NONE = "none";
-    public static final String ISBN = "isbn";
     private final S3Client s3Client;
     private final ResourceService resourceService;
-    private String brageRecordFile;
-    private List<Publication> publicationsToMerge;
-    private MergeSource source;
```

Comment:
Finally got rid of these two variables. I know why they were there, but I want them gone.

A function should either return a result OR modify environment, never both.